apps/prairielearn/src/server.ts

```
@@ -2035,7 +2041,14 @@ function idleErrorHandler(err: Error) {
   Sentry.close().finally(() => process.exit(1));
 }
 
-if (esMain(import.meta) && config.startServer) {
+if (!((esMain(import.meta) || import.meta.env?.DEV) && config.startServer)) {
+  exit(0);
+}
+let app: express.Express | undefined;
+
+// TODO: unindent this code alongside .git-blame-ignore-revs
+// eslint-disable-next-line no-constant-condition
+if (true) {
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)